src/app.fetch-httpClient.config.js

```diff
@@ -29,14 +35,16 @@ export class FetchConfig {
   get interceptor() {
     let auth    = this.auth;
     let config  = this.config;
+    let client = this.httpClient;
+    let storage = this.storage;
+    that = this;
 
     return {
       request(request) {
```

Comment:
I'm returning An Object with both request and response that's why curlys are needed